bin/azureauth

```diff
@@ -6,4 +6,4 @@
 set -e
 
 here=$(dirname $(readlink -f "$0"))
-"${here}/dotnet" run --project "${here}/../src/AzureAuth" -- $* --debug
\ No newline at end of file
+dotnet run --project "${here}/../src/AzureAuth" -- $* --debug
```

Comment:
It turns out it is difficult to test azureauth on it's own in development if you've just run the production version to warm the cache ;). The `dotnet` wrappers are still here to easily handle auth at build time to ensure dependencies are restored before using these scripts.